### PR TITLE
zh-update: /news/support/announcing-1.3-eol/index.md

### PR DESCRIPTION
**Pull request by yuxiaobo96**
ref:https://github.com/servicemesher/istio-official-translation/issues/1739

### CODE REVIEW

File: content/zh/news/support/announcing-1.3-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 对 Istio 1.3 的支持将于 2020 年 2 月 14 日终止 
+subtitle: 支持公告
+description: 即将到来的 Istio 1.3 生命周期终止公告。
+publishdate: 2020-01-15
+aliases:
+    - /zh/news/2020/announcing-1.3-eol
+---
+
+根据 Istio 的[支持策略](/about/release-cadence/)，在下一个 LTS 版本发布后，1.3 版本会继续得到三个月的支持。由于 [1.4 版本已于 11 月 14 日发布](/news/releases/1.4.x/announcing-1.4/)，因此对 1.3 版本的支持将于2020 年 2 月 14 日终止。
```
Comment (rootsongjc) [inline content/zh/news/support/announcing-1.3-eol/index.md]: 链家前加 `/zh` 前缀。

Comment (yuxiaobo96) [reply content/zh/news/support/announcing-1.3-eol/index.md]: ok

Comment (yuxiaobo96) [reply content/zh/news/support/announcing-1.3-eol/index.md]: done

File: content/zh/news/support/announcing-1.3-eol/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: 对 Istio 1.3 的支持将于 2020 年 2 月 14 日终止 
+subtitle: 支持公告
+description: 即将到来的 Istio 1.3 生命周期终止公告。
+publishdate: 2020-01-15
+aliases:
+    - /zh/news/2020/announcing-1.3-eol
+---
+
+根据 Istio 的[支持策略](/zh/about/release-cadence/)，在下一个 LTS 版本发布后，1.3 版本会继续得到三个月的支持。由于 [1.4 版本已于 11 月 14 日发布](/zh/news/releases/1.4.x/announcing-1.4/)，因此对 1.3 版本的支持将于2020 年 2 月 14 日终止。
```
Comment (rootsongjc) [inline content/zh/news/support/announcing-1.3-eol/index.md]: 2020 前少个空格。

**Pull request merged**